extensions/blocks/instagram-gallery/store/reducers.js

```diff
@@ -0,0 +1,10 @@
+export function tokens( state = {}, action ) {
+	switch ( action.type ) {
+		case 'INSTAGRAM_GALLERY_BLOCK_TOKEN_CONNECT':
+			return { ...state, [ action.token ]: 'connected' };
```

Comment:
Is it possible for there to be more than one token?

Reply:
Probably a very edge case, but it might happen if two users add the Instagram Gallery block (connected to themselves) to the same post.